status/flowcells.py

```diff
@@ -76,12 +78,123 @@ def list_flowcells(self, all=False):
 
         return OrderedDict(sorted(temp_flowcells.items(), reverse=True))
 
+    def list_ont_flowcells(self):
+        ont_flowcells = OrderedDict()
+        view_all = self.application.nanopore_runs_db.view("info/all_stats", descending=True)
+        view_status = self.application.nanopore_runs_db.view("info/run_status", descending=True)
+        view_project = self.application.projects_db.view("project/id_name_dates", descending=True)
+
+        for row in view_status:
+
+            if row.value == "finished":
+                name = row.key.split("/")[-1]
```

Comment:
`name` is same in both conditions

Reply:
94716065dfeec58e37a34b83db546801eaa55c64